src/os/bluestore/NVMEDevice.cc

```diff
@@ -385,8 +381,8 @@ void SharedDriverQueueData::_aio_handle(Task *t, IOContext *ioc)
       }
 
       t->queue = this;
-      lba_off = t->offset / sector_size;
-      lba_count = t->len / sector_size;
+      lba_off = t->offset / block_size;
```

Comment:
👍 but might want to move this change to another commit.

Reply:
Will split to two commits.